components/org.wso2.carbon.identity.oauth.uma.resource.endpoint/src/main/java/org/wso2/carbon/identity/oauth/uma/resource/endpoint/impl/ResourceRegistrationApiServiceImpl.java

```diff
@@ -42,16 +43,15 @@
 import java.util.regex.Pattern;
 import javax.ws.rs.core.Response;
 
-
 /**
  * ResourceRegistrationApiServiceImpl is used to handling resource management.
  */
 public class ResourceRegistrationApiServiceImpl extends ResourceRegistrationApiService {
 
     private static final Log log = LogFactory.getLog(ResourceRegistrationApiServiceImpl.class);
-    private static String tenantDomain = null;
-    private static String resourceOwnerName = null;
-    private static String consumerKey = null;
+    private static final String PAT_SCOPE = "uma_protection";
```

Comment:
Are these constants private to this class?
I mean won't these consants used else where as well?

Reply:
Yes. This is used only in this class .